### PR TITLE
Feat/artists contact

### PR DESCRIPTION
**Pull request by pdabrowsky**
https://app.asana.com/0/1202705605335915/1202705557275637/f

### CODE REVIEW

**Review by dominikx96 — CHANGES_REQUESTED** [review]
+ what's the valid phone number? I can't send the form 🤔 
+ make gaps between error and input smaller
Screenshot 2022-09-21 at 15 08 40

File: src/components/Contact/Contact.scss [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    top: calc(25px + 31px);
+    height: 489px;
+  }
+}
+
+.contact__form {
+  box-sizing: border-box;
+  padding: 16px;
+  width: calc(100% - 40px);
+  max-width: 504px;
```
[the remark this answers is not in the document]
Comment (pdabrowsky) [reply src/components/Contact/Contact.scss:78]: Valid number is like the example with` + `in the beginning `"+48112223344"`

File: src/components/Contact/Contact.scss [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  text-decoration: none;
+  font-size: 2.2rem;
+  font-weight: variables.$weight-bold-light;
+  line-height: 1.09;
+  @include breakpoints.small-desktop {
+    font-size: 2.6rem;
+  }
+}
+
+.contact_background {
```
Comment (dominikx96) [inline src/components/Contact/Contact.scss:63]: Should be 2 x `_`

File: src/components/Contact/Contact.scss [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  position: absolute;
+  z-index: -1;
+  object-fit: cover;
+  width: 100%;
+  //margin + header text line height
```
Comment (dominikx96) [inline src/components/Contact/Contact.scss:68]: make this type comments next to related line, not at the bottom

File: src/components/common/Textarea/Textarea.types.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+};
+
+export type AreaProps = {
+  label: string;
+  name: Path<FormData>;
+  register: UseFormRegister<FormData>;
+  required?: boolean;
+  classNames: ClassNames;
+  error?: string;
+};
```
Comment (dominikx96) [inline src/components/common/Textarea/Textarea.types.ts:18]: Name for this type should be `TextareaProps`, I also think we should follow pattern `ComponentNameXXX`

**Review by dominikx96 — CHANGES_REQUESTED** [review]
almost good!

File: src/components/Contact/Contact.tsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+        <p className="contact__info__desc">For business inquires or press related inquires please email us at:</p>
+        <a
+          href="mailto:info@nowness.com"
+          className="contact__info__email"
+        >
+          info@nowness.com
+        </a>
+      </address>
+      <form
+        onSubmit={handleSubmit((data) => handleContactSubmit(data))}
```
Comment (dominikx96) [inline src/components/Contact/Contact.tsx:46]: You can update to:

```
onSubmit={handleSubmit(handleContactSubmit)}
```
The `data` property still is availabe in the `handleContactSubmit` function :)

File: src/components/common/Input/Input.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+} as ComponentMeta<typeof Input>;
+
+export const InputStory: ComponentStory<typeof Input> = () => {
+  const { register } = useForm<FormData>();
+  return (
+    <Input
+      name="name"
+      label="Your name*"
+      placeholder="Evelyn Bennett"
+      classNames={{}}
```
Comment (dominikx96) [inline src/components/common/Input/Input.stories.tsx:21]: Make this prop optional, you shouldn't provide this prop if you don't need this

File: src/components/common/Input/Input.tsx [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  classNames,
+  name,
+  label,
+  placeholder = '',
+  required = true,
+  register,
+  error = '',
+  patern,
+}: InputProps) => {
+  const { wrapper = '', input = '', labelClass = '' } = classNames;
```
Comment (dominikx96) [inline src/components/common/Input/Input.tsx:15]: If `classNames` will be optional, you can make:

```
const { wrapper = '', input = '', labelClass = '' } = classNames || {}
```

File: src/components/common/Textarea/Textarea.stories.tsx [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+} as ComponentMeta<typeof Textarea>;
+
+export const TextareaStory: ComponentStory<typeof Textarea> = () => {
+  const { register } = useForm<FormData>();
+  return (
+    <Textarea
+      name="message"
+      label="Message*"
+      register={register}
+      classNames={{}}
```
Comment (dominikx96) [inline src/components/common/Textarea/Textarea.stories.tsx:21]: Same as above, do not provide an empty prop

File: src/components/common/Textarea/Textarea.tsx [hunk 9]
```diff
@@ -0,0 +1,29 @@
+import { TextareaProps } from './Textarea.types';
+
+import './Textarea.scss';
+
+const Textarea = ({ name, label, register, classNames, required = true, error = '' }: TextareaProps) => {
+  const { area = '', wrapper = '', labelClass = '' } = classNames;
```
Comment (dominikx96) [inline src/components/common/Textarea/Textarea.tsx:6]:
```
|| {}
``

**Review by dominikx96 — CHANGES_REQUESTED** [review]
almost good 💪

**Review by dominikx96 — APPROVED** [review]

**Pull request merged**